content/collections/flutter_sdk/en/flutter-sdk-4-0.md

```diff
@@ -68,44 +92,68 @@ class YourClass {
 
 ## Configure the SDK
 
-| Name  | Description | Default Value |
-| --- | --- | --- |
-| `apiKey` | `String`. The apiKey of your project. | `null` |
-| `flushQueueSize` | `int`. SDK attempts to upload once unsent event count exceeds the event upload threshold or reaches `flushIntervalMillis` interval.  | `30` |
-| `flushIntervalMillis` | `int`. The amount of time SDK attempts to upload the unsent events to the server or reaches the `flushQueueSize` threshold. The value is in milliseconds. | `30000` |
-| `instanceName` | `String`. The name of the instance. Instances with the same name shares storage and identity. For isolated storage and identity use a unique `instanceName` for each instance.  | `$default_instance`|
-| `optOut` | `bool`. Opt the user out of tracking. | `false` |
-| `logLevel` | `LogLevel` The log level. `LogLevel.off`, `LogLevel.error`, `LogLevel.warn`, `LogLevel.log`, `LogLevel.debug` | `LogLevel.warn` | 
-| `minIdLength` | `int`. The minimum length for user id or device id. | `5` |
-| `partnerId` | `int`. The partner id for partner integration. | `null` |
-| `flushMaxRetries` | `int`. Maximum retry times.  | `5` |
-| `useBatch` | `bool`. Whether to use batch API. | `false` |
-| `serverZone` | `ServerZone`. `ServerZone.us` or `ServerZone.eu`. The server zone to send to. Adjust server URL based on this config. | `ServerZone.us` |
-| `serverUrl` | `String`. The server URL events upload to. | `https://api2.amplitude.com/2/httpapi` |
-| `minTimeBetweenSessionsMillis` | `int`. The amount of time for session timeout. The value is in milliseconds. | `300000` |
-| `defaultTracking` | `DefaultTrackingOptions`. Options to control the default events tracking. | Check [Tracking default events](#tracking-default-events). |
```

Comment:
Is only defaultTracking removed in this block? Can we keep the original format so that it's easier to see what changes are included in this pr.